src/app/cards/components/search-page/search-page.component.html

```diff
@@ -3,6 +3,20 @@
   <mat-checkbox color="primary" formControlName="standard_legal">
     Standard Legal
   </mat-checkbox>
+  <mat-checkbox color="primary" formControlName="modern_legal">
+    Modern Legal
+  </mat-checkbox>
+  <mat-form-field>
+    <mat-label>Set</mat-label>
+    <mat-select formControlName="set">
+      <mat-option>
+        <ngx-mat-select-search [formControl]="setSearch" placeholderLabel="Search..."></ngx-mat-select-search>
```

Comment:
Don't want the dropdown filtering to be a part of the overall form, so create a separate FormControl object for it.